### ISSUE TITLE
Yield syntax

### ISSUE THREAD

**Issue by rac0316** [touilleMan/godot-python#168]
I would like to use the GDScript yield function in a .py file, but I'm not sure how to form the syntax or if it's supported.

GDScript syntax example:
`yield(get_tree(), "idle_frame")`
`yield(VisualServer, "frame_post_draw")`

Is this possible in Godot Python?

**Comment by touilleMan** [comment]
Hi @rac0316,
Yeld syntax for crossing bound between Godot and Python is not supported

I'm not sure it is even possible to implement this with the current GDnative API... Now is a good time to check this and provide some PRs for Godot 4 ;-)

**Comment by matheus2740** [comment]
Also I read somewhere that `yield` will be deprecated in favor of `await` in GDScript, so waiting for godot 4 sounds wise.

**Comment by filantus** [comment]
Yep, Godot 4 stable was released and there isn't `yield` available.
<img width="801" alt="img" src="https://user-images.githubusercontent.com/13046849/222585834-98cb21da-f9a5-4d3d-b31a-ea3d5e794ce8.png">
